common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -165,12 +168,48 @@ private boolean handleUrl(final WebView view, final String url) {
         } else if (isWebCpUrl(formattedURL)) {
             Logger.info(TAG, "It is a request from WebCP");
             return processWebCpRequest(view, url);
+        } else if (isPlayStoreUrl(formattedURL)){
+            Logger.info(TAG, "Request to open PlayStore.");
+            return processPlayStoreURL(view, url);
+        } else if (isAuthAppMFAUrl(formattedURL)){
+            Logger.info(TAG, "Request to link account with Authenticator.");
+            return processAuthAppMFAUrl(url);
+        } else if (isValidRedirectUri(url)){
+            Logger.info(TAG, "Check for Redirect Uri.");
+            return processRedirectUriCheck(view, url);
+        } else if (isBlankPageRequest(formattedURL)){
+            Logger.verbose(TAG, "It is an blank page request");
+            return true;
+        } else if (isUriSSLProtected(formattedURL)){
+            Logger.verbose(TAG, "Check for SSL protection");
+            return processSSLProtectionCheck(view, url);
         } else {
             Logger.info(TAG, "It is an invalid redirect uri.");
```

Comment:
OK, I hate this message.  Can we make it say something closer to what it means to us: "We have a URI that we know is not unsafe, but we don't have any special handling for.  This is potentially ADFS (or other cases that fall through here, which we should enumerate), deferring to the WebView for loading."